static/js/rich_search/stat_vars.tsx

```diff
@@ -31,21 +36,31 @@ function getURL(places: string[], statsVar: string): string {
   return `/tools/timeline#${params}`;
 }
 
-function StatVars({ places, statVars }: StatVarsPropType): JSX.Element {
-  if (!statVars.length) {
+export function StatVars({
```

Comment:
is this more appropriately named "StatVarResults"?

Reply:
Changed.